### ISSUE TITLE
Maven 0.5.1 release problem

### ISSUE THREAD

**Issue by Hronom** [nats-io/jnats#59]
The [artifact](http://search.maven.org/#artifactdetails%7Cio.nats%7Cjnats%7C0.5.1%7Cbundle):
```
 <dependencies>
    ...
    <dependency>
      <groupId>io.nats</groupId>
      <artifactId>jnats</artifactId>
      <version>0.5.1</version>
    </dependency>
  </dependencies>
```
can't be used because it have dependency on parent with `1.0-SNAPSHOT`, please fix this typo.

**Comment by oscerd** [comment]
Yes, this is related to https://github.com/nats-io/nats-parent-pom/issues/3

Maybe a release of parent is needed.

**Comment by mcqueary** [comment]
Hi -- yes, that was a mistake on my part. I just released `jnats-0.5.2`, which resolves this (depends on the stable `nats-parent-1.1.pom`). Give it a little while to show up on Central.

**Comment by oscerd** [comment]
Thanks Larry :-)

Il 29/ago/2016 20:49, "Larry McQueary" <notifications@github.com> ha
scritto:
